### PR TITLE
who is gus??

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

WHO IS HE!!

# Spriting

# Wiki Documentation

# Changelog

:cl:  

tweak: added a new line to the brain damage quirk 

/:cl:

### CODE REVIEW

**Review by ReddicusDragon — APPROVED** [review]
WHO IS GUS????

**Comment by cuackles** [comment]
I cannot put into words how much i love this PR

**Review by cuackles — APPROVED** [review]
Art approved cus this is art

**Comment by ktlwjec0** [comment]
I laughed out loud so I approve of this

**Review by Moltijoe — APPROVED** [review]
Aphie on blast

**Review by MajManatee — APPROVED** [review]
Approved because I was involved and he threatened to mcomplaint me

**Comment by solwashere** [comment]
Nobody here is answering my question

File: strings/traumas.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
 		"THESE NOOBS ARE RUINING MY FAVOURITE FUCKING SERVER.",
 		"AI not rogue, just had an ion law telling it to set shit on fire",
 		"CAN SPACE DRAGONS COMMIT MURDERS?",
-		"ok what? How do I keep getting this beard thing? ANd don't send me to mentor because they won't know"
+		"ok what? How do I keep getting this beard thing? ANd don't send me to mentor because they won't know",
+		"Who is Gus??? Tell me right now or else I'm about to get ugly"
```
Comment (SomeguyManperson) [inline strings/traumas.json]:
```suggestion
		"@pick(semicolon)Who is Gus??? Tell me right now or else I'm about to get ugly"
```

Comment (solwashere) [reply strings/traumas.json]: sure why not

**Comment by MajManatee** [comment]
theos just wanted his name on the PR

**Pull request merged**